pandas/core/internals/managers.py

```diff
@@ -1901,7 +1901,7 @@ def _compare_or_regex_match(a, b, regex=False):
     """
     Compare two array_like inputs of the same shape or two scalar values
```

Comment:
Since we moved from `match` to `search` here can you also updated the function name?

Reply:
OK.